### PR TITLE
fix: auto network switch modal

### PR DESCRIPTION
**Pull request by osslgtm**
## Summary

Modal on auto network switch, as discussed during sprint review

## Changes

networkSwitchModal in verify.tsx

## Issues

https://www.pivotaltracker.com/story/show/181325473

### CODE REVIEW

**Comment by osslgtm** [comment]
cc: @isaackps

File: src/common/contexts/network.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  const changeUserNetwork = useCallback(async () => {
+    if (currentChainId === currentChain) {
+      return;
+    }
+    const network: ChainId = currentChain;
+    try {
+      showOverlay(<LoadingModal title={"Changing Network..."} content={"Please respond to the metamask window"} />);
+      await changeNetwork(network);
+      console.log("changeNetwork");
+      console.log(currentChainId);
```
Comment (isaackps) [inline src/common/contexts/network.tsx]: please remove these.

**Review by isaackps — COMMENTED** [review]
there is a problem, once i click cancel or switch, it stuck on loading infinitely. please look into it

File: src/common/contexts/network.tsx (hunk 1 above)
Comment (osslgtm) [reply src/common/contexts/network.tsx]: paiseh i not done yet, set back to draft alr

**Comment by isaackps** [comment]
I refactored this story into another branch, PR #600

**Comment by isaackps** [comment]
PR #600 have been merged to master, thus closing this PR.

**Pull request closed without merging**